firmware/hw_layer/adc/adc_inputs.h

```diff
@@ -15,6 +15,13 @@
 #define SLOW_ADC_RATE 500
 #endif
 
+#if EFI_NON_UNIFORM_ANALOG_DIVIDER == FALSE
```

Comment:
I would prefer

```c++
__attribute__((weak)) float getAnalogInputDividerCoefficient(adc_channel_e) {
    return engineConfiguration->analogInputDividerCoefficient;
}
```

This will do the same thing, but requires no defines anywhere, just adding the function in the board's cpp file.

Reply:
Personally I don't like weak definition. As it sometimes cause strange issues that is hard to debug.
But, np, I'll rewrite it to use weak attribute.

Reply:
Hm, I've never had an issue with it like that. And in this case, LTO will inline it anyway when optimization is enabled.